app/scripts/directives/logViewer.js

```diff
@@ -401,6 +401,10 @@ angular.module('openshiftConsole')
                   return;
                 }
 
```

Comment:
Spelling of `metadata`:`context.project.metdata` vs `context.project.metadata`.

So most of our annotations are on a `filter` that, unfortunately, is now pulled in via a dependency from [origin-web-common](https://github.com/openshift/origin-web-common/blob/master/src/filters/annotation.js#L8).  We should probably add it there...

Reply:
@benjaminapetersen fixed the spelling

Reply:
thanks!  so by annotation, i mean making a [PR here](https://github.com/openshift/origin-web-common/blob/master/src/filters/annotation.js#L8) simply to add the line:

```JavaScript 
var annotationMap = {
  // truncated
  loggingKibanaUrl: ['openshift.io/logging.kibana.url']
}
```
And then when that PR merges, can update this PR to use the filter:

```JavaScript
url = "https://" + $scope.context.project.metadata.annotations[$filter('annotationName')('loggingKibanaUrl')];
```